_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,496 @@
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The text node automatically updates every 3 seconds after the page completes loading but it is not visible.
+
+```html
+<body onload="start()">
+	<p style="display: none">Random number: <span id="target">1</span></p>
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+	</script>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The text node automatically updates every 3 seconds but only as a result of an action from the user.
+
+```html
+<body>
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="start()" value="Start updates" />
+
+	<p>
+		The W3C Web Accessibility Initiative (WAI) develops standards and support materials to help you understand and
+		implement accessibility.
+	</p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			setInterval(change, 3000)
+		}
+	</script>
+</body>
+```
+
+#### Inapplicable Example 3
+
+The automatically updating content is the only content in the document.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stop()" value="Stop updates" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+[accessible name]: #accessible-name 'Definition of accessible name'
+[ascii whitespace]: https://infra.spec.whatwg.org/#ascii-whitespace
+[child]: https://dom.spec.whatwg.org/#concept-tree-child
+[content]: https://www.w3.org/TR/WCAG21/#dfn-content
+[document readiness]: https://www.w3.org/TR/html53/dom.html#current-document-readiness
+[essential]: https://www.w3.org/WAI/WCAG21/Understanding/pause-stop-hide.html#dfn-essential
+[flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree
+[f16]: https://www.w3.org/WAI/WCAG21/Techniques/failures/F16
+[g186]: https://www.w3.org/WAI/WCAG21/Techniques/general/G186
+[html document]: https://dom.spec.whatwg.org/#html-document
+[included in the accessibility tree]: #included-in-the-accessibility-tree 'Definition of included in the accessibility tree'
+[label]: https://www.w3.org/TR/WCAG21/#dfn-labels
+[mechanism]: https://www.w3.org/TR/WCAG21/#dfn-mechanism
```

Comment:
This definition is ambiguous. I don't think we can use this.

Reply:
It is being used in [audio or video that plays automatically has a control mechanism](https://act-rules.github.io/rules/4c31df)

Reply:
Not being used anymore in this rule